helpers/file_test.go

```diff
@@ -0,0 +1,87 @@
+package helpers_test
+
+import (
```

Comment:
Third party packages should be in the bottom

Reply:
💹  fixed in [11d19f5](https://github.com/carryall/go-google-scraper-challenge/pull/56/commits/11d19f544a42730c8e3df20961be177bcb577399)